src/effects/PitchEffect.js

```diff
@@ -52,7 +49,17 @@ class PitchEffect {
     * @returns {number} a playback ratio
     */
     getRatio (val) {
-        return this.tone.intervalToFrequencyRatio(val / 10);
+        return this.intervalToFrequencyRatio(val / 10);
+    }
+
+    /**
+    * Convert a musical interval to a frequency ratio.
+    * With thanks to Tone.js: https://github.com/Tonejs/Tone.js
+    * @param {number} interval - a musical interval, in semitones
+    * @returns {number} a frequency ratio
+    */
+    intervalToFrequencyRatio (interval) {
```

Comment:
Since this function is only used privately (right?) and is only called in one place, I'd say inline it (keeping the helpful comment about what it means) into `getRatio`